### PR TITLE
HOCS-5271: Allow a tag to be passed instead of calculating an increment

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: test/semver.test.js [hunk 1]
```diff
@@ -48,6 +49,17 @@ describe("isSemverIdentifier", () => {
   });
 });
 
+describe("parseTag", () => {
+  test("should allow all accepted tags", () => {
+    expect(parseTag("1.0.0")).toEqual("1.0.0");
+    expect(parseTag("1.0")).toEqual("1.0.0");
+    expect(parseTag("1")).toEqual("1.0.0");
+    expect(parseTag("1.0.0-0")).toEqual("1.0.0");
+    expect(parseTag("Sausages")).toEqual(undefined);
```
Comment (LiamMacP) [inline test/semver.test.js:58]: I love sausages

Comment (nickmcmahon01) [reply test/semver.test.js]: 🌭

File: src/index.js [hunk 2]
```diff
@@ -26,26 +27,29 @@ async function run() {
...
     }
 
     const octokit = getOctoKit(inputs.github_token);
     const repoTags = sortTags(await getTagsForRepo(octokit));
-    const newTag = generateSemverTag(
-      repoTags,
-      inputs.increment.toLowerCase(),
-      inputs.default_use_head_tag
-    );
+    const newTag = inputs.tag
```
Comment (LiamMacP) [inline src/index.js:46]:
```
    const newTag = inputs.tag ?? 
      generateSemverTag(
          repoTags,
          inputs.increment.toLowerCase(),
          inputs.default_use_head_tag
        );
```

**Review by LiamMacP — APPROVED** [review]
Nitpick change as will work.

**Pull request merged**